web/client/localConfig.json

```diff
@@ -681,5 +681,6 @@
           }
         ],
         "manager": ["Header", "Redirect", "Manager", "Home", "UserManager", "GroupManager", "Footer"]
-    }
+    },
+    "unsavedMapChangesDialog": true
```

Comment:
Please do not change localConfig when not needed.  
default values should be used elsewhere (default prop or in selector)
localConfig is used to override/customize default behaviors

- please add some documentation to the local-config.md for this new config property